src/pages/ReimbursementAccount/RequestorOnfidoStep.js

```diff
@@ -31,6 +31,16 @@ class RequestorOnfidoStep extends React.Component {
     constructor(props) {
         super(props);
         this.submit = this.submit.bind(this);
+        this.goBack = this.goBack.bind(this);
+    }
+
+    goBack(...args) {
+        const ondifyBack = document.getElementsByClassName('onfido-sdk-ui-NavigationBar-back');
```

Comment:
Can't we use `document.querySelector` to get the first occurrence of that class? As we won't be having the array this way code would be simplified

Reply:
Also typo in the const name `ondify` should be `onfido`

Reply:
@robertKozik yes you are right, fixed now.